R/importNet.MITAB.R

```diff
@@ -0,0 +1,141 @@
+OLOG <- as.character(getOption("rete.logfile"))
+logFileName(fPath = tempdir(), setOption = TRUE)
+logName <- unlist(getOption("rete.logfile"))
+if (file.exists(logName)) {file.remove(logName)}
+
+# ==== END SETUPS AND PREPARE ===================
+
+# ==== importNet.MITAB ==========================
+
+test_that("importNet.MITAB() rejects erroneus parameters", {
+  fN <- "core.psimitab"
+  
+  # === fName
+  
+  expect_error(
+    gG <- importNet.MITAB(fName = NULL,
+                          cutoffType = "xN",
+                          silent = TRUE),
+    '.checkArgs> "fName" mode error')
+
+  expect_error(
+    gG <- importNet.MITAB(fName = c("file_A", "file_B"),
+                          cutoffType = "xN",
+                          silent = TRUE),
+    '.checkArgs> "fName" length error')
+  
+  expect_error(
+    gG <- importNet.MITAB(fName = "noSuch.txt", 
+                          cutoffType = "xN",
+                          silent = TRUE),
+    '.checkArgs> "fName' error: file "noSuch.txt does not exist.")
+  
+  # ==== dropUnmapped
+  
+  expect_error(
+    gG <- importNet.MITAB(fName = fN,
+                          cutffType = "xN",
+                          dropUnmapped = NULL,
```

Comment:
Sorry, I wasn't clear enough. In the file  R/importPSIMITAB.R ,  the function signature doesn't have the parameter dropUnmapped.